packages/app-builder-lib/src/node-module-collector/types.ts

```diff
@@ -7,15 +7,12 @@ export interface NodeModuleInfo {
 
 export type ParsedDependencyTree = {
   readonly name: string
+  readonly from: string // for pnpm
```

Comment:
Would this be more appropriate in `PnpmDependency` since that extends `ParsedDependencyTree` already?

Reply:
Fixed.

Reply:
Fixed.